.env.example

```diff
@@ -64,6 +64,34 @@ LETSENCRYPT_STAGING=1
 #         "QFC_IS_LEGACY": false
 #     }
 # }
+
+# NOTE: for using a storage using WebDAV (NextCloud), that can be setup with:
+# {
+#     "webdav": {
+#         "BACKEND": "qfieldcloud.filestorage.backend.QfcWebDavStorage",
+#         "OPTIONS": {
+#             "webdav_url": "https://USERNAME:PASSWORD@my.webdav.server/remote.php/dav/files/USERNAME",
+#             "public_url": "https://my.webdav.server/public.php/webdav",
+#             "basic_auth": "NEXTCLOUD_SHARE_TOKEN:"
+#         },
+#         "QFC_IS_LEGACY": false
+#     }
+# }
+
+# Storage setup with `bytemark/WebDAV` docker image with:
+# NOTE: HTTP basic auth user and password must be the same as the WEBDAV_USERNAME and WEBDAV_PASSWORD env variables.
+# {
+#     "webdav": {
+#         "BACKEND": "qfieldcloud.filestorage.backend.QfcWebDavStorage",
+#         "OPTIONS": {
+#             "webdav_url": "http://qfc_webdav_user:qfc_webdav_pwd@172.17.0.1:8020",
+#             "public_url": "http://172.17.0.1:8020",
+#             "basic_auth": "qfc_webdav_user:qfc_webdav_pwd"
+#         },
+#         "QFC_IS_LEGACY": false
+#     }
+# }
```

Comment:
I am not 100% sure why we need these two examples? Probably you can change the keys to `webdav_nextcloud` in case we need to keep that one and change the leading comment for both cases to be similar?